app/komponenter/tekstBlokk/tekstBlokk.tsx

```diff
@@ -0,0 +1,44 @@
+import { TypografiTyper } from '~/typer/typografi';
+import React from 'react';
+import { PortableText } from '@portabletext/react';
+import { TypografiWrapper } from '~/utils/typografiWrapper';
+import css from './veilederhilsen.module.css';
+
+interface TekstBlokkProps {
+  tekstblokk: any | undefined;
+  valgBlock: string;
+  typografi?: TypografiTyper;
+}
+
+const TekstBlokk: React.FC<TekstBlokkProps> = ({
+  tekstblokk,
+  typografi,
+  valgBlock,
+}: TekstBlokkProps) => {
+  return tekstblokk ? (
+    <PortableText
+      value={tekstblokk[valgBlock]}
+      components={{
+        block: {
+          normal: ({ children }) => (
+            <TypografiWrapper typografi={typografi} style={{ margin: '0' }}>
+              {children}
+            </TypografiWrapper>
+          ),
+          h1: ({ children }) => (
+            <TypografiWrapper typografi={typografi}>
+              {children}
+            </TypografiWrapper>
+          ),
+          li: ({ children }) => (
+            <TypografiWrapper typografi={typografi}>
+              {children}
+            </TypografiWrapper>
+          ),
```

Comment:
Trenger vi dette? Jeg ville trodd at sanity fikset dette for oss og at vi ikke trenger custom logikk for bullet points, men kjenner ikke helt til det

Reply:
Enig, det er løst på en annen måte nå der hele punktlisten med overskriften og punkter ligger i sanity